doc/release-notes/0.51/0.51.md

```diff
@@ -137,6 +137,27 @@ on x64 platforms, but builds with OpenJDK 17 and later also require more stack s
 <td valign="top">Avoid using the <tt>-XX:+ShowHiddenFrames</tt> option with OpenJDK 18 and later.</td>
 </tr>
 
+<tr>
+<td valign="top">N/A </td>
+<td valign="top">JVM fails to load AOT compiled code from portable shared class cache created on IBM z17.</td>
+<td valign="top">Linux on IBM Z®</td>
+<td valign="top">When docker container image with IBM Semeru Runtime is created
```

Comment:
Pls replace IBM Semeru Runtime with Eclipse OpenJ9. These are release notes for Eclipse OpenJ9, while Semeru is the build IBM creates from Eclipse OpenJ9.